src/workflows/dags/app_notifications.py

```diff
@@ -179,6 +179,37 @@ def send_notifications(messages):
     notifications = create_notifications(recipients)
     send_notifications(notifications)
 
+@dag(
+    "Push-Notifications",
```

Comment:
Hi @BenjaminSsempala , thanks for this. Much appreciated as notifications play a significant role in our vision of personalisation within the mobile app.

I am curious about how we are approaching `broadcast` notifications......

Based on user feedback patterns, many people have mixed feelings about scheduled `broadcast` notifications, especially if they are not "`personalised enough`".

As a team, have we had any opportunity to have any high level conceptual discussions on this subject of mobile notifications?